src/main/kotlin/com/github/blarc/ai/commits/intellij/plugin/notifications/Notification.kt

```diff
@@ -44,6 +44,7 @@ data class Notification(
         fun unsuccessfulRequest(message: String) = Notification(message = message("notifications.unsuccessful-request", message))
         fun noCommitMessage() = Notification(message = message("notifications.no-commit-message"))
         fun unableToSaveToken(message: String?) = Notification(message = message("notifications.unable-to-save-token", message ?: "Unknown error"))
+        fun unableToRetrieveToken(message: String?) = Notification(message = message("notifications.unable-to-retrieve-token", message ?: "Unknown error"))
```

Comment:
Where is this method used? If it's not used, please delete it and the corresponding message.